src/mqtt/config/sensormqtt.xml

```diff
@@ -30,10 +30,12 @@
         <!-- POSTPROCESSOR -->
         <postprocessors>
 			
+			<!-- This postprocessor will bypass whatever comes in the payload to the upper layers -->
```

Comment:
It would be so much easier to read this if the indentation were only 4 spaces ...